### PR TITLE
add io.github.appoutlet.GameOutlet

### PR DESCRIPTION
**Pull request by MessiasLima**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: flathub.json
Comment (hfiguiere) [inline flathub.json:1]: you don't need this file.

File: io.github.appoutlet.GameOutlet.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+app-id: io.github.appoutlet.GameOutlet
+runtime: org.gnome.Platform
```
Comment (hfiguiere) [inline io.github.appoutlet.GameOutlet.yaml:2]: do you really need the GNOME platform?

File: io.github.appoutlet.GameOutlet.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: GameOutlet
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline io.github.appoutlet.GameOutlet.yaml:11]: if it supports wayland then line 10 is `fallback-x11`. if it doesn't, then line 11 should be removed.

File: io.github.appoutlet.GameOutlet.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-desktop
+  - --socket=session-bus
```
Comment (hfiguiere) [inline io.github.appoutlet.GameOutlet.yaml:15]: line 15: this is not allowed. you have to determine which dbus name you want to talk to.

File: io.github.appoutlet.GameOutlet.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+command: GameOutlet
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+finish-args:
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-desktop
```
Comment (hfiguiere) [inline io.github.appoutlet.GameOutlet.yaml:14]: why do you need access to xdg-desktop?

File: io.github.appoutlet.GameOutlet.yaml (hunk 1 above)
Comment (MessiasLima) [reply io.github.appoutlet.GameOutlet.yaml:2]: I think so. I'm using `Desktop.getDesktop().browse(URI(url))` to open links in browser. Under the hood, this line checks the supported actions. I found this internal class
 
Screenshot from 2023-06-18 15-17-10

When I use `org.freedesktop.Sdk` I've got "The BROWSE action is not supported in your platform"

File: flathub.json
Comment (MessiasLima) [reply flathub.json:1]: Deleted [f09d4f1](https://github.com/flathub/flathub/pull/4254/commits/f09d4f1702aba84879a1e1ec9eb15fbe337bfe69)

**Comment by hfiguiere** [comment]
bot, build io.github.appoutlet.GameOutlet

**Comment by MessiasLima** [comment]
@hfiguiere Thank you for your quick response 🚀

File: io.github.appoutlet.GameOutlet.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    <mediatype>application/x-jar</mediatype>
+  </provides>
+  <keywords>
+    <keyword>Games</keyword>
+    <keyword>Gaming</keyword>
+    <keyword>Deals</keyword>
+    <keyword>Compose</keyword>
+    <keyword>Kotlin</keyword>
+  </keywords>
+</component>
```
Comment (hfiguiere) [inline io.github.appoutlet.GameOutlet.metainfo.xml:56]:
```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

Comment (MessiasLima) [reply io.github.appoutlet.GameOutlet.metainfo.xml:57]: Added here https://github.com/flathub/flathub/pull/4254/commits/67bf9af7209316addc83531099b93c3e21fa194d

**Comment by MessiasLima** [comment]
bot, build io.github.appoutlet.GameOutlet

**Pull request closed without merging**